spec/models/ems_event_spec.rb

```diff
@@ -102,6 +102,62 @@ def mock_raw_event_vm(raw_event)
     end
   end
 
+  context ".process_container_entities_in_event!" do
+    before :each do
+      @ems = FactoryGirl.create(:ems_kubernetes)
+      @container_project = FactoryGirl.create(:container_project, :ext_management_system => @ems)
+      EMS_REF = "test_ems_ref"
+      @event_hash = {
+        :ems_ref => EMS_REF,
+        :ems_id  => @ems.id
+      }
+    end
+
+    context "process node in events" do
+      before :each do
```

Comment:
> I think it would be better to move the initialize of the objects into the test itself, by that you can also remove the contexts, unless more tests are coming?

@alongoldboim embrace yourself, more tests are coming :) 
for post refresh linking, after the manageiq timestamp addition is merged #6265